flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/utils/AggFunctionFactory.scala

```diff
@@ -105,6 +105,13 @@ class AggFunctionFactory(
           throw new TableException("CUME_DIST Function is not supported in stream mode.")
         }
 
+      case a: SqlRankFunction if a.getKind == SqlKind.PERCENT_RANK =>
+        if (isBounded) {
+          createPercentRankAggFunction(argTypes)
+        } else {
+          throw new TableException("PERCENT_RANK Function is not supported in batch mode.")
```

Comment:
`is not supported in stream mode` ?

Reply:
Yes,  `PERCENT_RANK` need to know the size of window partition. But in stream mode, we can't know the size as the data is unbounded.

Reply:
Yeah, I mean the exception cue is 'batch mode'.

Reply:
Thanks for pointing it.